crates/node/src/cli.rs

```diff
@@ -15,7 +15,6 @@ pub enum Sealing {
 /// Available testnets.
 #[derive(Debug, Copy, Clone, PartialEq, clap::ValueEnum)]
 pub enum Testnet {
-    Local,
```

Comment:
No more local testnet? How come?

Reply:
We never used the flag `--testnet local`, the `--chain local` works as usual